onboarding.md

```diff
@@ -61,52 +60,78 @@ This is a checklist for onboarding new members, workers, and contractors to the
 1. Add member SIN to Passbolt and share access with `People Operations` group.
```

Comment:
Add member SIN to BitWarden... (all mentions of Passbolt should be changed to BitWarden since we changed tools)

Reply:
I just changed the text. Thought it would be easier but hope that's the right process